odim-controller/scripts/gen_odimra_certs.sh

```diff
@@ -626,6 +632,57 @@ validate_config_params()
 	fi
 }
 
+generate_etcd_certs()
+{
+	CommonName=etcd
+	SAN_DNS_LIST="\
+	DNS.1 = etcd"
+
+	if [[ ! -e ${ODIMRA_ROOTCA_CRT_PATH} ]] || [[ ! -e ${ODIMRA_ROOTCA_KEY_PATH} ]]; then
+		echo "[$(date)] -- ERROR -- root CA files not present in ${ODIMRA_ROOTCA_CRT_PATH}"
+		exit 1
+	fi
+
+	#generate etcd private key
+	${OPENSSL_BIN_PATH} genrsa -out ${ODIMRA_ETCD_SERVER_KEY_PATH} ${KEY_LENGTH}
+	eval_cmd_exec $? "${ODIMRA_ETCD_SERVER_KEY_PATH} generation failed"
+
+	#generate etcd csr
+	${OPENSSL_BIN_PATH} req -new -sha512 -key ${ODIMRA_ETCD_SERVER_KEY_PATH} -subj "/C=IN/ST=KA/O=HPE/CN=${CommonName}" -config <(cat <<EOF
```

Comment:
Make use below definitions for distinguished names also instead of "/C=IN/ST=KA/O=HPE"
CERT_COUNTRY_NAME="US"
CERT_STATE_NAME="CA"
CERT_LOCALITY_NAME="California"
CERT_ORGANIZATION_NAME="HPE"
CERT_ORGANIZATION_UNIT_NAME="Telco Solutions"

Reply:
addressed